handler/notification_server_handler.go

```diff
@@ -66,30 +67,28 @@ func NewNgsiV2SubscriptionHandler(receivers ...NotificationReceiver) Handler {
 }
 
 func NgsiV2SubscriptionHandler(receivers []NotificationReceiver, w http.ResponseWriter, r *http.Request) error {
-	if r.Method != "POST" {
-		return StatusError{http.StatusMethodNotAllowed, errors.New("Expected a POST")}
+	if r.Method != http.MethodPost {
+		return StatusError{http.StatusMethodNotAllowed, fmt.Errorf("expected method POST, got %s", r.Method)}
 	}
 
 	if ct := r.Header.Get("Content-Type"); ct != "" {
 		if !strings.HasPrefix(ct, "application/json") {
-			return StatusError{http.StatusBadRequest, errors.New("Invalid notification payload")}
+			return StatusError{Code: http.StatusBadRequest, Err: errors.New("invalid notification payload")}
 		}
 	}
 
 	// maximum read of 8MB, the current max for Orion (https://fiware-orion.readthedocs.io/en/master/user/known_limitations/index.html)
-	r.Body = http.MaxBytesReader(w, r.Body, 8*1024*1024)
-
-	decoder := json.NewDecoder(r.Body)
+	r.Body = http.MaxBytesReader(w, r.Body, 8<<20)
```

Comment:
What is the purpose of using the bitwise operation instead of the numeric multiplication?

Reply:
I my opinion it is better to use bit shifting for multiplication of power of two, even if finally is the same result but I think we should use a const for readability. What do you think about it?